Lib/fontTools/feaLib/ast.py

```diff
@@ -319,21 +319,20 @@ def addDefinition(self, definition):
         assert isinstance(definition, MarkClassDefinition)
         self.definitions.append(definition)
         for glyph in definition.glyphSet():
-            if glyph in self.definitions:
-                otherLoc = self.definitions[glyph].location
-                raise FeatureLibError(
-                    "Glyph %s already defined at %s:%d:%d" % (
-                        glyph, otherLoc[0], otherLoc[1], otherLoc[2]),
-                    definition.location)
+# Fixing this code fails tests!
```

Comment:
The code branch had a bug in it (it was looking in the wrong list for the glyph) and once I fixed that, some tests were failing because they were declaring markclass information for the same glyph more than once. I didn't want to have to make that decision so I commented out the branch since previously it had been, in effect, doing nothing.